src/infra/core/src/engine/engine_spark.rs

```diff
@@ -324,7 +324,7 @@ impl SparkEngine {
 
         {
             tracing::info!(?request, path = ?request_path, "Writing request");
-            let file = File::create(&request_path)?;
+            let file: File = File::create(&request_path)?;
```

Comment:
Why was this necessary?

Reply:
It's enough to make 1 unexpected click in VSCode to insert a type, and visually it looks almost the same as an auto hint